packages/dhis2/src/Adaptor.js

```diff
@@ -82,7 +82,7 @@ axios.interceptors.response.use(
           responseData: response.data,
         };
 
-        Log.error(newError.message);
+        console.error(`✗ Error at ${new Date()}:\n∟`, newError.message);
```

Comment:
i think that printing the date is now redundant.

Reply:
Fixed https://github.com/OpenFn/adaptors/pull/417/commits/6b89e24bb01f87c94ec229072927becfb976d56f